### ISSUE TITLE
NodalDisplacement lists duplicate information

### ISSUE THREAD

**Issue by xRadne** [strusoft/femdesign-api#461]
NodalDisplacement lists duplicate information (see node 9 below). This may cause confusion. I think we should remove duplicate info in the `NodalDisplacement` component in Grasshopper. 

There might be other results that behave in the same way.

```TSV
Nodal displacements, Load comb.: ULS
ID	Node	ex	ey	ez	fix	fiy	fiz	Comb.
[-]	[-]	[m]	[m]	[m]	[rad]	[rad]	[rad]	[-]
Bottom.1.1	9	-0.013	0.000	-0.042	0.000	0.016	0.000	ULS
Bottom.1.1	7	-0.013	0.000	-0.042	0.000	0.016	0.000	ULS
Bottom.1.1	8	-0.013	0.000	-0.042	0.000	0.016	0.000	ULS
Bottom.2.1	9	-0.013	0.000	-0.042	0.000	0.016	0.000	ULS
```

**Comment by andosca** [comment]
@xRadne Do you mean that it is listing the same node, `9`, twice?

**Comment by xRadne** [comment]
Yes, and I understand that it's for convenience for a human reading the `.csv`file. But I think it is inconvenient when creating a script to map the nodal displacent results to each FEA bar object as an example.

I think it is possible to limit the results to only one row per node using the options in the bsc files.

**Comment by andosca** [comment]
I think this might be a question for BP. When listing `nodal displacements`, I agree, you are expecting nodal displacements and not any information about which bar or shell that node belongs to. If you are interested in bar och shell displacements there are `bar displacement` and `shell displacement` results. In general I am an opponent of culling or modifying the data from FEM-Design - I would rather let the user do that. So I think it is worthwhile investigating why BP is listing the information like this first.

**Comment by andosca** [comment]
https://strusoft.fogbugz.com/f/cases/121840/Nodal-displacements

**Comment by andosca** [comment]
BP will continue to group by object causing this to happen.
